PubHub/PubHub.Common/ErrorSpecifications/ConflictError.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PubHub.Common.ErrorSpecifications
+{
+    public static class ConflictError
+    {
+        public const string TITLE = "Conflicting values";
+        public const string ERROR_MESSAGE = $"We couldn't processs your action. Try again or contact PubHub support if the problem persists";
+    }
```

Comment:
This should have an error code associated, like the others (:

Reply:
I do not know what the error code for this one is.

Reply:
Yesa. We currently have no grouping for errors (: